doc/releases/migration-guide-3.7.rst

```diff
@@ -120,6 +120,10 @@ General Purpose I/O (GPIO)
 Input
 =====
 
+* The ``analog-axis`` deadzone calibration value has been changed to be
+  realtive to the raw adc values, similarly to min and max. The data structures
+  and properties have been renamed accordingly.
```

Comment:
Since this is the migration guide (i.e not just "Release notes"), could this be worded in a way that makes it extra clear what people need to actually do to adapt their app w.r.t. the proposed change?

Reply:
yeah good point, added a note about the fact that the value should be scaled